doc/data-structures.rst

```diff
@@ -393,14 +393,14 @@ methods (like pandas) for transforming datasets into new objects.
 
 For removing variables, you can select and drop an explicit list of
```

Comment:
should we add a "tip" box saying that the old way of using `drop` still works but we encourage use of the more explicit `drop_vars`?

Reply:
I think the docs should have the recommended approach only; there's no need to have legacy methods there. i.e. if the answer to "Is there any reason to use `drop` for new code" is 'No', no need to put it in the docs. 

It's still in the API documentation and the `whatsnew`, if anyone is reviewing old code.

Open-minded as ever though!

Reply:
:+1: